per/drf_views.py

```diff
@@ -383,14 +377,14 @@ def get(self, request, pk, format=None):
         )
         if assessment_queryset.exists():
             for assessent in assessment_queryset.first().area_responses.all():
```

Comment:
```suggestion
        if assessent := assessment_queryset.first():
            for area_response in assessent.area_responses.all():
```


Let's use Prefetch for all nested items here, we can use filters as well
https://docs.djangoproject.com/en/3.2/ref/models/querysets/#prefetch-related
```python
>>> vegetarian_pizzas = Pizza.objects.filter(vegetarian=True)
>>> Restaurant.objects.prefetch_related(
...     Prefetch("pizzas", queryset=vegetarian_pizzas, to_attr="vegetarian_menu"),
...     "vegetarian_menu__toppings",
... )
>>> [menu for menu in restaurant_obj.vegetarian_menu] # Not database fetching is needed
```